tdms/src/iterator.cpp

```diff
@@ -6408,12 +6325,9 @@ if(runmode == rm_complete && (nvertices>0) )
     freeCastMatlab3DArray(KsourceR,((int)(J1[0]-J0[0]+1.)));
   }
 
-if( !((N_fieldsample_i == 0) || (N_fieldsample_j == 0) || (N_fieldsample_k == 0) || (N_fieldsample_n == 0)) ){
-  freeCastMatlab4DArray( fieldsample, N_fieldsample_k,N_fieldsample_n);
-  freeCastMatlab4DArray( fieldsample_x, N_fieldsample_k,N_fieldsample_n);
-  freeCastMatlab4DArray( fieldsample_y, N_fieldsample_k,N_fieldsample_n);
-  freeCastMatlab4DArray( fieldsample_z, N_fieldsample_k,N_fieldsample_n);

```

Comment:
Why were these removed?

Reply:
`fieldsample_x` etc. are now not allocated (assuming #52 means they can be deleted)